### PR TITLE
Fix/tr 260 261 266 fix strings for cognia

### PR DESCRIPTION
**Pull request by lecosson**
Related to:
- https://oat-sa.atlassian.net/browse/TR-260
- https://oat-sa.atlassian.net/browse/TR-261
- https://oat-sa.atlassian.net/browse/TR-266

Replace some strings for Cognia 

Related PR's:
- https://github.com/oat-sa/extension-tao-mp/pull/509

### CODE REVIEW

**Comment by meefox** [comment]
is it still draft?

**Review by meefox — APPROVED** [review]

File: models/translation/ManualTranslations.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+class ManualTranslations
+{
+    public function getManualTranslations()
+    {
+        return [
+            __('End test'),
+            __('Skip and end test'),
+            __('Submit the test'),
+            __('SUBMIT THE TEST'),
```
Comment (wazelin) [inline models/translation/ManualTranslations.php:30]: Wait what?
Why do you need manual translations for those, especially on the core extension?

They should **already** be in the QTI test extension or a test runner extension.
https://github.com/oat-sa/extension-tao-testqti/blob/master/locales/en-US/messages.po#L1212

**Review by wazelin — CHANGES_REQUESTED** [review]

**Comment by wazelin** [comment]
Consider closing the PR, I don't think it's needed.

File: models/translation/ManualTranslations.php (hunk 1 above)
Comment (btamas) [reply models/translation/ManualTranslations.php:30]: I totally agree. I do not understand the aim of this PR.

**Pull request closed without merging**